tests/test_skills/test_error.py

```diff
@@ -181,14 +179,6 @@ def test_error_unsupported_skill(self):
         assert msg.get("type") == DefaultMessage.Type.ERROR
         assert msg.get("error_code") == DefaultMessage.ErrorCode.UNSUPPORTED_SKILL.value
 
-    def test_error_behaviour_instantiation(self):
```

Comment:
why this has been removed?

Also, maybe we could get rid of the `ErrorBehaviour` anyway

Reply:
We have gotten rid of the `ErrorBehaviour`, hence this has been removed.